timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -181,4 +193,24 @@ services:
         http: POST /sw/{namespace}
         args:
           namespace: string
-          request: LockWatchRequest
\ No newline at end of file
+          request: LockWatchRequest
+  CrossClientBatchedConjureTimelockService:
```

Comment:
missed on pass 1: I don't think CrossClientBatched should be a part of the service name as it's more of an implementation detail, it should really be something like MultiClientConjureTimelockService?